docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -68,7 +72,7 @@ If the same problem occurs with this test two more times, the test run finishes
 
 The `browser-list-comma-separated` argument specifies the list of browsers (separated by commas) where tests are run.
 
-> You can specify different browser types in one command call.
+*Overrides a configuration file property*: [browsers](configuration-file.md#browsers).
```

Comment:
These messages are difficult to distinguish from the main text, in this file they are in the end of each paragraph but [here](https://github.com/DevExpress/testcafe/blob/4de7432773293fa82c9daaf649403e1e8387beaf/docs/articles/documentation/using-testcafe/programming-interface/runner.md) they placed in  different way.
Maybe we should use smth like NOTE style for them

Reply:
I didn't want to make them too noticeable. We need a simple cross-reference here.
The NOTE style brings too much graphics - icon, borders, italics... - in each section. I don't want so much attention.

As to where they are place in CLI and JS API - this is dictated by the way they correspond to each other.

In CLI, one flag corresponds to one option in most cases. So, each cross-reference is placed at the end of the corresponding section.

In API, some config options correspond to one API method. In this case, they are placed at the end of the section, like in CLI. Other config options correspond to a method's argument. In this case, I could place them inside the table in a new column. I've tried this, but it makes tables too wide and it introduces inconsistency. So, I ended up with lists at the end of each section.

Reply:
So, that's what my logic is. We can fix it if we come up with any counterarguments. \cc @helen-dikareva

Reply:
ok, I see you really have thought this through.